README.md

```diff
@@ -76,10 +76,10 @@ Specifically, what if we gave the Post model a `category_name` attribute?
 
 ## You can define your own attributes on models
 
-Since our ActiveRecord models are still just Ruby classes, we can define our own set
+Since our ActiveRecord models are still just Ruby classes, we can define our own setter
```

Comment:
go into more detail here on why you'd want to define a `category_name` method on the `Post` model - and why we're defining a customer setter as opposed to adding a column to the db

I would also rename the header on line 75, maybe something that alludes to the purpose behind modifying a model